wallet/src/App.vue

```diff
@@ -129,7 +133,49 @@ export default defineComponent({
     handleLogOut() {
       console.log("Wallet Disconnected");
       this.walletAddress = "";
-      this.setWalletType(WalletType.NONE);
+      this.walletStore.setWalletType(WalletType.NONE);
+    },
+    async executeTx(amount: number) {
+      try {
+        const webMode = this.walletStore.getWebMode;
+        const txParams: types.ExecParams[] = [
+          {
+            type: types.TransactionType.TransferAlgo,
+            sign: types.SignType.SecretKey,
+            fromAccount: { addr: this.walletAddress, sk: new Uint8Array(0) },
+            toAccountAddr: toAddress,
+            amountMicroAlgos: amount,
+            payFlags: { totalFee: 1000 },
+          },
+        ];
+        let response = await webMode.executeTx(txParams);
+        this.transactionMessage =
+          amount / 1e6 +
+          " Algos has been trasnferred on this address: " +
+          toAddress;
+        console.log(response);
+      } catch (error) {
+        console.warn(error);
+      }
+    },
+    async executeAppTx() {
+      try {
+        const webMode = this.walletStore.getWebMode;
+        const execParams: types.ExecParams[] = [
+          {
+            type: types.TransactionType.CallApp,
+            sign: types.SignType.SecretKey,
+            fromAccount: { addr: this.walletAddress, sk: new Uint8Array(0) },
+            appID: 189,
```

Comment:
Why appID is constant here?

Reply:
applicationID has been added as parameter similar to amount in above method for sending algos.